scooby/report.py

```diff
@@ -70,6 +72,19 @@ def total_ram(self):
     def date(self):
         return time.strftime('%a %b %d %H:%M:%S %Y %Z')
 
+    @property
+    def filesystem(self):
+        """Get the type of the file system at the path of the scooby package"""
+        # Code by https://stackoverflow.com/a/35291824/10504481
+        mypath = str(Path(__file__).resolve())
+        bestMatch = ""
+        fsType = ""
+        for part in psutil.disk_partitions():
+            if mypath.startswith(part.mountpoint) and len(bestMatch) < len(part.mountpoint):
+                fsType = part.fstype
+                bestMatch = part.mountpoint
+        return fsType
```

Comment:
Would it be OK to change `fsType` to `fs_type` and `bestMatch` to `best_match`, avoiding camelCase. I see that the linked stackoverflow answer uses this, but I think it is OK to adjust it to the style used in Scooby.

Reply:
Fixed in a48b7323d3f9e34fb7bb1d8d9c783db0971713d2